scipy/sparse/construct.py

```diff
@@ -791,5 +791,20 @@ def rand(m, n, density=0.01, format="coo", dtype=None, random_state=None):
     -----
     Only float types are supported for now.
 
+    See Also
+    --------
+    scipy.sparse.random : Equivalent function when passing `data_rvs=None`
```

Comment:
That is what I thought. Therefore, this is the docstring of `rand` pointing to `random`. Is that what you meant?